jail-buddy.lic

```diff
@@ -54,7 +54,12 @@ def get_out_of_jail(old_loc)
   Script.running.find_all { |s| !s.paused? && !s.no_pause_all }.each(&:pause)
   stop_script('go2') if Script.running?('go2')
   echo('** DON\'T PANIC JAIL BUDDY HAS YOUR BACK! **')
+  fput('stand') if not standing?
+  start_script('outdoorsmanship',['10 "dust bunny"'])
```

Comment:
💡 I _think_ you can make this a bit clearer with `start_script('outdoorsmanship', [10, "dust bunny"])`

Reply:
That's what I tried first, but start_script didn't parse the arguments correctly.  This way worked

Reply:
Sounds good